lib/util/paging-iterator.js

```diff
@@ -81,11 +83,13 @@ class PagingIterator {
 			this.nextField = PAGING_MODES.MARKER;
 			this.nextValue = data.next_marker;
 		} else {
-			throw new Error('Unable to determine paging strategy for response!');
+			// throw new Error('Unable to determine paging strategy for response!');
+			this.nextField = PAGING_MODES.MARKER;
+			this.nextValue = null;
+			this.done = true;
```

Comment:
Why do you need this here?

Wouldn't it get set to `true` in the same way as if `data.next_marker` were `null` in the `else if` block?